### ISSUE TITLE
Evaluate doesn't support chinese string

### ISSUE THREAD

**Issue by biantech** [jpmml/jpmml-evaluator#117]
I have a PMML model , SEG6-PMMLv1-Modified-F-Model.xml , when i run it in SAS, the score result is correct. But I use JPmml to evaluate the value , the evaluate score is not correct. I don't know why?
The score value display in "PROB1" column in Data file is correct. but JPmml can't evaluate such score values. 

[doc.zip](https://github.com/jpmml/jpmml-evaluator/files/2086626/doc.zip)
Can anyone debug the issues.  
please use utf-8 to read the file.

**Comment by vruusmann** [comment]
The JPMML-Evaluator library is carrying out computations exactly as specified in the PMML file.

I can identify two issues with the sample PMML file. First, in your dataset, the constant `-999999` probably denotes a missing value? The PMML file does not contain information about it, and therefore JPMML-Evaluator treats those values as regular numeric values. To correct the problem, the PMML file could/should contain the following helper markup:
```XML
<DataField name="LN_PMT_SCHEDULED_AMT_SUM_AUTO" displayName="ln_pmt_scheduled_amt_sum_auto" optype="continuous" dataType="double">
  <Value value="-999999"/ property="missing"> <!-- THIS -->
</DataField>
```

Second, the categorical field `STUDY_RESULT` does not specify valid category values - they are all empty strings:
```XML
<DataField name="STUDY_RESULT" displayName="study_result" optype="categorical" dataType="string">
  <Value value="" property="valid"/>
  <Value value="" property="valid"/>
</DataField>
```

As a result, all `STUDY_RESULT` input values are converted to missing values (on the basis of `<MiningField name="STUDY_RESULT" invalidValueTreatment="asMissing" missingValueTreatment="asIs"/>`), and evaluated as such.

In conclusion, the SPM has generated an incorrect PMML file. Your claim that SAS is able to generate correct predictions with it seems dubious to me.
